cloud/endagaweb/settings/prod.py

```diff
@@ -293,9 +293,18 @@
     }
 }
 
-EMAIL_BACKEND = 'django_mailgun.MailgunBackend'
-MAILGUN_ACCESS_KEY = os.environ.get("MAILGUN_ACCESS_KEY", 'key-testkeypleaseignore')
-MAILGUN_SERVER_NAME = os.environ.get("MAILGUN_SERVER_NAME", '')
+# Email settings
```

Comment:
It would be preferable to keep the default option (Mailgun) and add the gmail settings as a comment.